### ISSUE TITLE
train val accuracy is not as high as mentioned. Plus res101 accuracy curve is not stable

### ISSUE THREAD

**Issue by 1093842024** [ooooverflow/BiSeNet#7]
i train with context_path: resnet101 and resnet 18.
      first question: both the validation accuracy hardly reach 0.9, Mostly stop as 0.88-0.89.
      second question: while the resnet101 training, the validation accuracy will fluctuate a lot after epoch 100, and will drop a lot after about epoch 180+.
      could you please share your training parameter like lr,batchsize,GPU num,crop_height,crop_width and some detailed trick?

**Comment by hubutui** [comment]
Try another optimizer? The authors use SGD, and this repo use RMSprop, not sure why. Why not have a try with Adam, or just the simple SGD with momentum >= 0.9, and adjust your learning rate according to learning curve in tensorboardx.

**Comment by JunjieZhouwust** [comment]
Which dataset did you train? I train resnet101 with CamVid, train loss is close to 0.3, precision_val is close to 0.25 after 35 epoch. Do you know what is the problem?
